tests/fixtures/responseWithNoResults.json

```diff
@@ -0,0 +1,2228 @@
+{
+  "verticalResults": {
+    "appliedQueryFilters": [],
+    "results": [],
+    "resultsCount": 0,
+    "source": "KNOWLEDGE_MANAGER"
+  },
+  "queryId": "94c572c7-8fa5-4435-a956-7c13667af173",
+  "searchIntents": [],
+  "facets": [],
+  "locationBias": {
+    "latitude": 43.6349,
+    "longitude": -116.2023,
```

Comment:
I would maybe fudge these numbers a little bit if you haven't already

Reply:
hahah true, i think this is in Boise somewhere! I'll do a passthrough and replace things with cleaned data